framer/Context.coffee

```diff
@@ -50,6 +50,10 @@ class exports.Context extends BaseClass
 		@_parent = options.parent
 		@_name = options.name
 		
+		@perspective = 1200
```

Comment:
What happens if you chance these now. Nothing right? Are they supposed to be static?